draft-janfred-radext-radiusdtls-bis.md

```diff
@@ -569,10 +572,7 @@ Any security-related issue causes the connection to be closed.
 After security restrictions have been applied, any unexpected traffic may be safely ignored, as it cannot cause a security issue.
 This allows for future extensions to the RADIUS/DTLS specifications.
 
-Once a DTLS session is established, a RADIUS/DTLS server SHOULD use DTLS Heartbeats {{!RFC6520}} to determine connectivity between the two servers.
-A server SHOULD also use watchdog packets from the client to determine that the session is still active.
-
-As UDP does not guarantee delivery of messages, RADIUS/DTLS servers that do not implement an application-layer watchdog MUST also maintain a "Last Traffic" timestamp per DTLS session.
+As UDP does not guarantee delivery of messages, RADIUS/DTLS servers MUST maintain a "Last Traffic" timestamp per DTLS session.
```

Comment:
The spec regarding DTLS Heartbeats and application-layer watchdog is now redundant, since we have the requirement in the section above.
Maybe we should reference it here, but at least the text should be removed.

However, for a server there must still be a timer for deleting the DTLS session. AFAIK the server was completely passive before, especially since with RAIDUS/UDP there was no need to save state for the client, if there were no packets sent for a while.